### PR TITLE
fix of base64 jpg image

### PR DESCRIPTION
**Pull request by calogero93**
### What this PR does 📖
- fix base64 jpg image crash

### Which issue(s) this PR fixes 🔨
- Resolve #

### CODE REVIEW

File: src/components/ui_kit/photo_picker/mod.rs [hunk 1]
```diff
@@ -49,8 +49,14 @@ pub fn PhotoPicker(cx: Scope<Props>) -> Element {
                     let path_image = FileDialog::new().add_filter("image", &["jpg", "png", "jpeg"]).set_directory(".").pick_file();
                     match path_image {
                         Some(path) => {
-                            let base64_image = to_base64(path.to_str().unwrap());
-                            match account.write().update_identity(IdentityUpdate::set_graphics_picture(base64_image)) {
+                            
+                            let file = std::fs::read(&path).unwrap();
+
+                            let prefix = String::from("data:image/png;base64,");
```
Comment (dariusc93) [inline src/components/ui_kit/photo_picker/mod.rs]: Might want try grabbing either the extension of the file or checking the header of it (latter may be preferred).

File: src/components/ui_kit/photo_picker/mod.rs [hunk 2]
```diff
@@ -49,8 +49,14 @@ pub fn PhotoPicker(cx: Scope<Props>) -> Element {
                     let path_image = FileDialog::new().add_filter("image", &["jpg", "png", "jpeg"]).set_directory(".").pick_file();
                     match path_image {
                         Some(path) => {
-                            let base64_image = to_base64(path.to_str().unwrap());
-                            match account.write().update_identity(IdentityUpdate::set_graphics_picture(base64_image)) {
+                            
+                            let file = std::fs::read(&path).unwrap();
```
Comment (dariusc93) [inline src/components/ui_kit/photo_picker/mod.rs]: I wouldnt recommend unwrapping. Could use a match condition to be on the safe side.

**Review by dariusc93 — COMMENTED** [review]
Left a couple of comments.

**Comment by calogero93** [comment]
@dariusc93 I updated this PR can you have a look at this and leave other comments if needed? Thanks

**Pull request merged**